packages/tpetra/core/example/Finite-Element-Assembly/fem_assembly_InsertGlobalIndices_DP.cpp

```diff
@@ -58,19 +58,17 @@
 
 using namespace TpetraExamples;
 
+#define COMM_T Teuchos::RCP<const Teuchos::Comm<int> >
```

Comment:
Please don't do this (define a short, non-namespaced macro).  Make this a typedef, not a macro:
```
using comm_ptr_type = Teuchos::RCP<const Teuchos::Comm<int> >;
```